portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio - Interest</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+
+      <!-- navigation bar -->
+      <nav class="w3-bar w3-black">
+        <a href="index.html" class="w3-bar-item w3-button">Home</a>
+        <a href="experience.html" class="w3-bar-item w3-button">Experience</a>
+        <a href="/" class="w3-bar-item w3-button">Interest</a>
```

Comment:
should it be `interest.html` here and `/` below? :)

Reply:
thank you  for spotting this!